### PR TITLE
[CHORE] Updated Message Logs

### LINKED ISSUE #828: [CHORE] - Update Message Logs
### What work needs to be performed?

Update the message deletion and message edit logs to include the channel name, and the edit logs to include the message link.

To complete this:

- In [this file](https://github.com/BeccaLyria/discord-bot/blob/main/src/events/messageEvents/messageUpdate.ts) add a field to the `updateEmbed` with the name `Channel` and the value `<#${newMessage.channel.id}>`.
- Add another field to that `updateEmbed` with the name `Message link` and the value `newMessage.url`.
- In [this file](https://github.com/BeccaLyria/discord-bot/blob/main/src/events/messageEvents/messageDelete.ts) add a field to the `deleteEmbed` with the name `Channel` and the value `<#${channel.id}>`.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Akshu-on-github**
# Pull Request

## Description

This PR modifies the files messageUpdate.ts and messageDelete.ts

## Related Issue

Closes #828

## Documentation

For _any_ version updates, please verify if the [documentation page](https://www.beccalyria.com/discord-documentation) needs an update. If it does, please [create an issue there](https://github.com/BeccaLyria/discord-documentation/issues/new?assignees=nhcarrigan&labels=%F0%9F%9A%A6+status%3A+awaiting+triage&template=update.md&title=%5BUPDATE%5D) to ensure it is not forgotten.

- [x] My contribution does NOT require a documentation update.
- [ ] My contribution DOES require a documentation update.

### CODE REVIEW

File: src/events/messageEvents/messageDelete.ts [hunk 1]
```diff
@@ -29,6 +29,7 @@ export const messageDelete = async (
     deleteEmbed.setDescription("Here is my record of that message.");
     deleteEmbed.addField("Channel", `<#${channel.id}>`);
     deleteEmbed.setTimestamp();
+    deleteEmbed.channel(`<#${channel.id}>`);
```
Comment (nhcarrigan) [inline src/events/messageEvents/messageDelete.ts:32]:
```suggestion
```

Comment (nhcarrigan) [reply src/events/messageEvents/messageDelete.ts:32]: I was wrong, the delete embed already includes the channel.

File: src/events/messageEvents/messageUpdate.ts [hunk 2]
```diff
@@ -47,6 +47,8 @@ export const messageUpdate = async (
     updateEmbed.setFooter(`Author: ${author.id} | Message: ${oldMessage.id}`);
     updateEmbed.setColor(Becca.colours.default);
     updateEmbed.setTimestamp();
+    updateEmbed.channel(`<#${newMessage.channel.id}>`);
```
Comment (nhcarrigan) [inline src/events/messageEvents/messageUpdate.ts:50]:
```suggestion
    updateEmbed.addField("Channel", `<#${newMessage.channel.id}>`);
```

File: src/events/messageEvents/messageUpdate.ts [hunk 3]
```diff
@@ -47,6 +47,8 @@ export const messageUpdate = async (
...
+    updateEmbed.messageLink(newMessage.url)
```
Comment (nhcarrigan) [inline src/events/messageEvents/messageUpdate.ts:51]:
```suggestion
    updateEmbed.addField("Message Link", newMessage.url);
```

**Review by nhcarrigan — CHANGES_REQUESTED** [review]
Couple of quick things.

**Comment by Akshu-on-github** [comment]
All good now I hope?

**Review by nhcarrigan — APPROVED** [review]
Looks good to me!

**Comment by nhcarrigan** [comment]
## Congratulations

Congratulations on making your first pull request to our project. We have reviewed your contributions and are happy to accept them.

We hope you enjoyed the contribution process, and encourage you to watch for issues labelled `help wanted` for additional opportunities to contribute.

**Pull request merged**